src/modules/auth/auth.controller.ts

```diff
@@ -81,4 +85,22 @@ export class AuthController {
   async verifyOtp(@Body() authVerifyOtpDto: AuthVerifyOtpDto): Promise<void> {
     return this.authService.verifyOtp(authVerifyOtpDto);
   }
+
+  @PublicRoute()
+  @HttpCode(200)
+  @Post('restore-password')
+  async restorePassword(
+    @Body() authRestorePasswordOtpCode: AuthRestorePasswordDto,
+  ): Promise<void> {
+    return this.authService.restorePassword(authRestorePasswordOtpCode);
+  }
+
+  @PublicRoute()
```

Comment:
usually everyone do oposite => public by default and protected should be defined. you will have too many public routes in this project to each time sign them @public. or at least check, maybe you can make controller (instead of endpoint) public, so you can avoid sign each endpoint

Reply:
I did it in [SB-79](https://github.com/ZenBit-Tech/sunny-bunnies_be/pull/36)
Im avoid @PublicRoute for each endpoint.